src/Microsoft.Azure.WebJobs.Host/Timers/TaskSeriesTimer.cs

```diff
@@ -21,7 +21,7 @@ internal sealed class TaskSeriesTimer : ITaskSeriesTimer
         private Task _run;
         private bool _disposed;
 
-        public TaskSeriesTimer(ITaskSeriesCommand command, IBackgroundExceptionDispatcher backgroundExceptionDispatcher,
+        public TaskSeriesTimer(ITaskSeriesCommand command, IWebJobExceptionHandler backgroundExceptionDispatcher,
```

Comment:
Yeah, there's a bunch of these. Even though it's mostly search/replace, I didn't want to have to re-do it, so I figured I'd wait until we finalized it.

---
In reply to: [76664040](https://github.com/Azure/azure-webjobs-sdk/pull/799#discussion_r76664040) [](ancestors = 76664040)